server/migrations/admin-migration.js

```diff
@@ -0,0 +1,28 @@
+'use strict';
+/** @type {import('sequelize-cli').Migration} */
```

Comment:
You want to have a sequential number as part of the migration file name, something like:
003-admin-migration.js